### PR TITLE
Add UPM registry action

### PR DESCRIPTION
**Pull request by homapf**
Adding the possibility to add new registries in the project manifest with a composite action.

### CODE REVIEW

File: .github/actions/add-registry/add-upm-registry.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+# Define the path to the manifest file
+manifest_file="Packages/manifest.json"
+
+# Check if the registry already exists in the manifest file
+if grep -q "$url" "$manifest_file"; then
+  echo "Registry $name already exists in manifest file."
+else
+  # Add the registry to the manifest file
+  jq '.scopedRegistries += [{"name":"'"$name"'","url":"'"$url"'","scopes":["'"$scope"'"]}]' $manifest_file > tmp.$$.json && mv tmp.$$.json $manifest_file
```
Comment (manelizzard) [inline .github/actions/add-registry/add-upm-registry.sh:16]: Does `jq` work on Mac OS machines out of the box?

**Review by manelizzard — COMMENTED** [review]
Smart! Just one question

File: .github/actions/add-registry/add-upm-registry.sh (hunk 1 above)
Comment (homapf) [reply .github/actions/add-registry/add-upm-registry.sh:16]: I don't think so, but I guess when using this on mac we will have control over the machine so we can install manually ?

Comment (manelizzard) [reply .github/actions/add-registry/add-upm-registry.sh:16]: Yeah, that's probably right. 👍

**Review by manelizzard — APPROVED** [review]

**Pull request merged**